.gitignore

```diff
@@ -3,4 +3,4 @@ www/index.md
 www/dist
 /node_modules
 /web_modules
-__tests__/**/*/web_modules
+__tests__/integration/*/web_modules
```

Comment:
I also ended up doing some cleanup of the test files. We had some test-generated files being added in git. Sorry for the spam!